io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AbstractAnalyseTable.java

```diff
@@ -71,7 +72,9 @@ protected void analyse(LocatableInputSupplier<? extends InputStream> source, boo
 			evaluator = wb.getCreationHelper().createFormulaEvaluator();
 
 			// the first row represents the header
-			analyseHeader(sheet);
+			if (skipNlines <= 0) {
```

Comment:
Negate the condition?

Reply:
If no lines are to be skipped that means there is no header to be analysed because the first row is already instance data.

Reply:
If we want to skip one line, then we assume that first row is the header, but in that case `analyseHeader` is not called at all anymore.
The case with multiple lines to skip is harder to assess, that was obviously not in the original concept of this implementation. In line with what was done before would be to analyse the first line in this case as well.

Reply:
For me makes no sense to analyse the header if anyway is not going to be used for the case with skipNlines > 0.